Source/Core/CylinderGeometry.js

```diff
@@ -78,18 +76,10 @@ define([
         var slices = defaultValue(options.slices, 128);
 
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(length)) {
-            throw new DeveloperError('options.length must be defined.');
-        }
-        if (!defined(topRadius)) {
-            throw new DeveloperError('options.topRadius must be defined.');
-        }
-        if (!defined(bottomRadius)) {
-            throw new DeveloperError('options.bottomRadius must be defined.');
-        }
-        if (slices < 3) {
-            throw new DeveloperError('options.slices must be greater than or equal to 3.');
-        }
+        Check.defined('length', length);
+        Check.defined('topRadius', topRadius);
+        Check.defined('bottomRadius', bottomRadius);
+        Check.number.greaterThanOrEquals('slices', slices, 3);
```

Comment:
Yeah, looks like you accidentally have some changes from your other branches here.  Do you need to merge in master?